malboxes/templates/win10_32_analyst.json

```diff
@@ -22,8 +22,6 @@
 		]
 	}],
 
-	{% include 'snippets/postprocessor_vagrant.json' %},
-
```

Comment:
why is this removed? This looks like a mistake.

Reply:
Oh, it looks like it was there two times by mistake... Wouldn't removing this affect the ESX use-case?